wondrous-app/components/profile/modals/index.tsx

```diff
@@ -64,26 +80,32 @@ const UserItem = (props) => {
         }}
       >
         <CommentTopFlexDiv>
-          <TaskSubmissionHeaderCreatorText
+          <NameText
             style={{
               marginBottom: '4px',
             }}
           >
             {user?.username}
-          </TaskSubmissionHeaderCreatorText>
+          </NameText>
         </CommentTopFlexDiv>
-        <CommentText>{cutString(user?.bio)}</CommentText>
+        <CommentLine>{cutString(user?.bio)}</CommentLine>
       </div>
     </UserWrapper>
   );
 };
+
 export const MoreInfoModal = (props) => {
   const { orgId, podId, showUsers, showPods, open, handleClose, name } = props;
   const [displayUsers, setDisplayUsers] = useState(showUsers);
   const [listLoading, setListLoading] = useState(true);
   const [displayPods, setDisplayPods] = useState(showPods);
   const [userList, setUserList] = useState([]);
   const [podList, setPodList] = useState([]);
+  const [userListSearch, setUserListSearch] = useState([]);
+  const [podListSearch, setPodListSearch] = useState([]);
+  const [dividedUserList, setDividedUserList] = useState([]);
+  const [dividedPodList, setDividedPodList] = useState([]);
+  const [activeTab, setActiveTab] = useState('contributors');
```

Comment:
can you create a config for these vars? e.g TABS_MAP = {CONTRIBUTORS: 'contributors', POD: 'pod'} etc

Reply:
done